media/renderers/BUILD.gn

```diff
@@ -2,11 +2,18 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+if (is_cobalt) {
+  import("//starboard/build/buildflags.gni")
+}
+
 source_set("renderers") {
   # Do not expand the visibility here without double-checking with OWNERS, this
   # is a roll-up target which is part of the //media component. Most other DEPs
   # should be using //media and not directly DEP this roll-up target.
   visibility = [ "//media" ]
+  if (is_cobalt && use_starboard_media) {
```

Comment:
I think `use_starboard_media` may be redundant here, because `is_cobalt` always implies it, no ?

Reply:
Right now `use_starboard_media` can be set independently of `is_cobalt`, if we want to get rid of it entirely let's have a separate conversation if we want to get rid of it